docs/chymyst04.md

```diff
@@ -205,16 +205,18 @@ val x = f() // fetch the current value
 Also note that each invocation of `makeCounter` will create new, fresh molecules `counter`, `decr`, and `fetch` inside the closure, because each invocation will create a fresh local scope.
 In this way, the user can create as many independent counters as desired.
 
-This example shows how we can “hide” some molecules and yet use their reactions.
-A closure can define local reaction with several input molecules, inject some of these molecules initially, and return some (but not all) molecule constructors to the global scope outside of the closure.
+This example shows how we can encapsulate some molecules and yet use their reactions.
+A closure can define local reaction with several input molecules, inject some of these molecules initially, and return some (but not all) molecule injectors to 
+the 
+global scope outside of the closure.
 
 # Example: Concurrent merge-sort
 
-Chemical laws can be “recursive”: a molecule can start a reaction whose reaction body defines further reactions and injects the same molecule.
+Chemical laws can be recursive: a molecule can start a reaction whose reaction body defines further reactions and injects the same molecule.
 Since each reaction body will have a fresh scope, new molecules and new reactions will be defined every time.
-This will create a recursive configuration of new reactions, such as a linked list or a tree of reactions.
+This will create a recursive configuration of reactions, such as a linked list or a tree of reactions.
 
-We will now figure out how to use “recursive chemistry” for implementing the merge-sort algorithm in `JoinRun`.
+We will now figure out how to use **recursive chemistry** for implementing the merge-sort algorithm in `JoinRun`.
```

Comment:
Here I agree that quotes are not necessary, but I would also not make the boldface **recursive chemistry**.  I reserve boldface for definitions of important new terms. Here I would just say "recursive molecules" (without the quotes) instead of "recursive chemistry".